legal/security_research.md

```diff
@@ -0,0 +1,60 @@
+# Policies for Security/Community Research
+
+This document lays out the Julia projects policies and guidance for researchers
+wishing to perform research on Julia's software development processes,
+contributors or other aspects of the julia community,
+
+For security issues in the software itself, please see the [Security policy](https://github.com/JuliaLang/julia/security/policy)
+or email [security@julialang.org](security@julialang.org).
+
+## Scope
+
+This document applies to the Julia programming language itself, the Julia package registries, as well as any packages directly affiliated with the Julia project. *Additionally, wherever this policy imposes any notification, disclosure or other requirements, such requirements also apply for research involving any packages registered in the Julia package registry (and thus distributed by the Julia project)*.
```

Comment:
Can we clarify what "packages directly affiliated with the Julia project" means? My package is built using Julia, is it affiliated?

Reply:
Well, I meant to include packages that are sort of being developed along with Julia (e.g
 JuliaDebug, JuliaArrays) and thus would inherit policies, while excluding things in this clause that are clearly not (e.g. JuMP). Not sure how to best word that without giving an explicit list that we don't have. I suppose we could just have packages add a statement that they are covered by JuliaLang policies (CoC, security policies, this , etc)

Reply:
The ColPrac contributors guide includes the text:

>Interactions with people in the community must always follow the community standards[link],
> including in pull requests, issues, and discussions.

It could very reasonable also include the text:

> Security research involving this package, should be inline with the JuliaLang Policies for Security/Community Research[link]

I think other Contributors Guides/CONTRIBUTING.md/READMEs could reasonably include something similar